src/theme/Navbar.tsx

```diff
@@ -6,7 +6,7 @@ import ThemeSwitch from '../components/ThemeSwitch'
 
 const Navbar: FC = () => {
   return (
-    <nav className="Navbar fixed left-0 right-0 z-nav flex w-screen justify-center bg-light-surface-1 dark:border-dark-surface-3 dark:bg-dark-surface-1">
+    <nav className="Navbar navbar fixed left-0 right-0 z-nav flex w-screen justify-center bg-light-surface-1 dark:border-dark-surface-3 dark:bg-dark-surface-1">
```

Comment:
added this since docusaurus use 'navbar' classname to get the nav bar on docs page!